docs/administration/reasoning.md

```diff
@@ -1,7 +1,172 @@
+## Inference rules
 
-    We are doing our best to complete this page. 
-    If you want to participae, dont hesitate to join the [Filigran Community on Slack](https://community.filigran.io) 
-    or submit your pull request on the [Github doc repository](https://github.com/OpenCTI-Platform/docs).
+The rules engine comprises a set of predefined rules (named inference rules) that govern how new relationships are inferred based on existing data. These rules are carefully crafted to ensure logical and accurate relationship creation. Here is the list of existing inference rules:
+
+#### Raise incident based on sighting
+
+| Conditions                                      | Creations                                                                       |
+|:------------------------------------------------|:--------------------------------------------------------------------------------|
+| A non-revoked Indicator is sighted in an Entity | Creation of an Incident linked to the sighted Indicator and the targeted Entity |
+
+![Incident_based_on_sighting](assets/incident_on_sighting.png)
+
+#### Sightings of observables via observed data
+
+| Conditions                                                           | Creations                                                                                   |
+|:---------------------------------------------------------------------|:--------------------------------------------------------------------------------------------|
+| An Indicator is based on an Observable contained in an Observed Data | Creation of a sighting between the Indicator and the creating Identity of the Observed Data |
+
+![Sighting_via_observed_data](assets/sighting_via_obs_data.png)
+
+#### Sightings propagation from indicator
+
+| Conditions                                                  | Creations                               |
+|:------------------------------------------------------------|:----------------------------------------|
+| An Indicator based on an Observable is sighted in an Entity | The Observable is sighted in the Entity |
+
+![Sighting_prop_from_indicator](assets/sighting_from_indicator.png)
+
+#### Sightings propagation from observable
+
+| Conditions                                                  | Creations                              |
+|:------------------------------------------------------------|:---------------------------------------|
+| An Indicator is based on an Observable sighted in an Entity | The Indicator is sighted in the Entity |
+
+![Sighting_prop_from_observable](assets/sighting_from_observable.png)
+
+#### Relation propagation via an observable
+
+| Conditions                                 | Creations                                                  |
+|:-------------------------------------------|:-----------------------------------------------------------|
+| An observable is _related to_ two Entities | Create a _relate to_ relationship between the two Entities |
```

Comment:
```suggestion
| An observable is _related to_ two Entities | Create a _related to_ relationship between the two Entities |
```
I'm not sure, but just in case